src/plugins/betterExperiments/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { Button, FluxDispatcher, GuildStore } from "@webpack/common";
+
+const ExperimentStore = findStoreLazy("ExperimentStore");
+// const GuildTooltip = findByCodeLazy("GuildTooltip");
+const GuildIcon = findByCodeLazy(".PureComponent){render(){return(0,");
+// const GuildIcon = wreq(565138).Z;
+// .PureComponent){render(){return(0,
+export default definePlugin({
+    authors: [Devs.mantikafasi],
+    name: "BetterExperiments",
+    description: "makes guild experiments look better",
+
+    patches: [
+        {
+            find: "Guild Assignments",
+            replacement: {
+                match: /Guild Assignments"}\),\(0,.\.jsx.+?}\)/,
+                replace: "Guild Assignments\"}),$self.getExperimentsComponent(e)"
+            },
+        },
+        // stole from experiments plugin troll
+        {
+            find: 'H1,title:"Experiments"',
+            replacement: {
+                match: 'title:"Experiments",children:[',
+                replace: "$&$self.refreshButton(),"
+            }
+        },
+    ],
+    settings: definePluginSettings({
+
+    }),
+    refreshButton: () => {
+        return <Button
+            size={Button.Sizes.LARGE}
+            color={Button.Colors.PRIMARY}
+            look={Button.Looks.FILLED}
+            onClick={() => {
+                FluxDispatcher.dispatch({ type: "EXPERIMENTS_FETCH", withGuildExperiments: true });
+            }}
+            // TODO should probably move these to css file
+            style={{
+                marginTop: "4px",
+                marginBottom: "8px",
+                fontSize: "16px",
+            }}
+        >
+            Refresh Experiments
+        </Button>;
+    },
+    getExperimentsComponent: (e: any) => {
+        const experiments: Object = ExperimentStore.getRegisteredExperiments();
+
+        const guildIds = GuildStore.getGuildIds();
+        const guilds = GuildStore.getGuilds();
+        const exp = experiments[e.experimentId];
+        // bucket to description map
+        const bucketMap = {};
+        if (!exp.buckets) return;
+
+        for (const [i, v] of exp.buckets.entries()) {
+            bucketMap[v] = exp.description[i];
+        }
+
+        return <div>
+            <div key={e.experimentId} className="vc-guild-experiment">
+                {Object.keys(bucketMap).map(bucket => {
+                    const description = bucketMap[bucket];
+                    const guildIcons = guildIds.map(guildId => {
+                        const descriptor = ExperimentStore.getGuildExperimentDescriptor(e.experimentId, guildId);
+                        const guild = guilds[guildId];
+                        if (!descriptor) return null;
+                        if (descriptor.bucket === parseInt(bucket)) return (
+                            <GuildIcon showTooltip={true} showBadge={true} guild={guild} key={guild.id} />
+                        );
+                    }).filter(Boolean);
```

Comment:
is this a new featurfe
```suggestion
                    }).filter(Boolean);
```